go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/containerd/containerd v1.7.8
 	github.com/containerd/continuity v0.4.3
 	github.com/containerd/log v0.1.0
+	github.com/containerd/nerdctl v1.7.0
```

Comment:
I think this creates a circular dependency in nerdctl. Maybe we should see if we can get the rootless util package in a separate module.